docs/user/exams/instructors_guide.rst

```diff
@@ -307,6 +307,7 @@ If you want you can also enable the :ref:`second correction <Before the exam's a
 
 - To assess a submission a second time go to the exercise assessment dashboard. When it is enabled, a |start_new_assessment| button will be visible in the second correction round.
 - The new second assessment will have all the feedback copied from the first assessment. Those can be overridden, and new feedback can be added as well. This does not override the original result, but saves a separate second result.
+- Within the second correction round assessment instructors and tutors can highlight which feedback was created for which correction round. This is displayed as a batch at the bottom of every feedback. This view can be enabled or disabled any time during a second correction round assessment. The feature is currently possible for text, modeling and fileupload exercises.
```

Comment:
```suggestion
- Within the second correction round assessment instructors and tutors can highlight which feedback was created for which correction round. This is displayed as a badge at the bottom of every feedback. This view can be enabled or disabled any time during the second correction by/with.... The feature is currently possible for text, modeling and fileupload exercises.
```
Also see suggestion from Dominik. I would add especially how users can toggle this feature.

Reply:
done